client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.html

```diff
@@ -112,20 +112,19 @@
                     <mat-cell (click)="openEditInfo(motion, $event)" *matCellDef="let motion">
                         <div class="fill">
                             <div class="innerTable state-column">
-                                <div class="small ellipsis-overflow" *ngIf="motion.category">
-                                    <mat-icon>device_hub</mat-icon>
-                                    {{ motion.category }}
+                                <div class="ellipsis-overflow" *ngIf="motion.category">
+                                    <os-icon-container icon="device_hub">{{ motion.category }}</os-icon-container>
                                 </div>
-                                <div class="small ellipsis-overflow" *ngIf="motion.motion_block">
-                                    <mat-icon>widgets</mat-icon>
-                                    {{ motion.motion_block.title }}
+                                <div class="ellipsis-overflow" *ngIf="motion.motion_block">
```

Comment:
Please add more margin-top between icons/lines in meta column (e.g. use 'spacer-top-5' class).
Also in other list views.